<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SenderLog2Files</class>
 <widget class="QWidget" name="SenderLog2Files">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Работа с файлами *.log2</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelNumberChannel1">
         <property name="text">
          <string>№ канала:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxCh1">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bttnChooseFiles1">
         <property name="text">
          <string>Выбрать файлы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar1">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCh1">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Зациклить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelNumberChannel2">
         <property name="text">
          <string>№ канала:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxCh2">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bttnChooseFiles2">
         <property name="text">
          <string>Выбрать файлы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar2">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCh2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Зациклить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelNumberChannel3">
         <property name="text">
          <string>№ канала:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxCh3">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bttnChooseFiles3">
         <property name="text">
          <string>Выбрать файлы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar3">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCh3">
         <property name="text">
          <string>Зациклить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="labelNumberChannel4">
         <property name="text">
          <string>№ канала:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxCh4">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bttnChooseFiles4">
         <property name="text">
          <string>Выбрать файлы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar4">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCh4">
         <property name="text">
          <string>Зациклить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="labelNumberChannel5">
         <property name="text">
          <string>№ канала:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxCh5">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bttnChooseFiles5">
         <property name="text">
          <string>Выбрать файлы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar5">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCh5">
         <property name="text">
          <string>Зациклить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="bttnStartSending">
       <property name="text">
        <string>Начать передачу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bttnStopSending">
       <property name="text">
        <string>Остановить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
